velox/exec/GroupingSet.cpp

```diff
@@ -299,6 +305,59 @@ void GroupingSet::initializeGlobalAggregation() {
   globalAggregationInitialized_ = true;
 }
 
+void GroupingSet::addAggregationInputWithoutGrouping(
+    const RowVectorPtr& input,
+    bool mayPushdown) {
+  activeRows_.resize(input->size());
+  activeRows_.setAll();
+
+  // CreateHashTable() initializes aggregates_. i.e set offsets
+  if (not table_) {
+    createHashTable();
+  }
+  // Copy over the keys
+  auto& hashers = table_->hashers();
+  if (ignoreNullKeys_) {
+    deselectRowsWithNulls(hashers, activeRows_);
+  }
+
+  const vector_size_t input_size = activeRows_.countSelected();
+  char* groups[input_size];
```

Comment:
Will this uses up the stack space?  Maybe using a buffer allocated by memory pool is safer.